StratisCore.UI/src/app/shared/pipes/seconds-to-string.pipe.ts

```diff
@@ -0,0 +1,48 @@
+import { Pipe, PipeTransform } from '@angular/core';
+
+@Pipe({
+  name: 'secondsToString'
+})
+export class SecondsToStringPipe implements PipeTransform {
+  constructor() {
+  }
+
+  public transform(seconds: number): string {
+    let numDays = Math.floor(seconds / 86400);
+    let numHours = Math.floor((seconds % 86400) / 3600);
+    let numMinutes = Math.floor(((seconds % 86400) % 3600) / 60);
+    let numSeconds = ((seconds % 86400) % 3600) % 60;
+    let dateString = "";
+
+    if (numDays > 0) {
+      if (numDays > 1) {
+        dateString += numDays + " days ";
+      } else {
+        dateString += numDays + " day ";
+      }
+    }
+
+    if (numHours > 0) {
+      if (numHours > 1) {
```

Comment:
could simplify nesting by using just 
```
dateString += `${numDays} day${numDays > 1 ? 's' : ''} `;
```

or
```
private toPlural(value: number) {
    return value > 1 ? 's' : '';
}

dateString += `${numDays} day${toPlural(numDays)} `;
```

and apply it in this class

Reply:
This was existing code that I moved from dashboard.cshtml but yes I can improve